clients/unshaded/src/main/java/tachyon/client/block/RemoteBlockOutStream.java

```diff
@@ -45,6 +45,22 @@ public RemoteBlockOutStream(long blockId, long blockSize) throws IOException {
         mWorkerClient.getSessionId());
   }
 
+  /**
+   * Creates a new block output stream on a specific host.
+   *
+   * @param blockId the block id
+   * @param blockSize the block size
+   * @param hostname the hostname of the preferred worker
+   * @throws IOException if I/O error occurs
+   */
+  public RemoteBlockOutStream(long blockId, long blockSize, String hostname) throws IOException {
+    super(blockId, blockSize);
+    mRemoteWriter = RemoteBlockWriter.Factory.createRemoteBlockWriter(ClientContext.getConf());
+    mWorkerClient = mContext.acquireWorkerClient(hostname);
+    mRemoteWriter.open(mWorkerClient.getDataServerAddress(), mBlockId,
+        mWorkerClient.getSessionId());
```

Comment:
does this need to be on a new line? (it seems like it would fit the 100 character limit)